sw/device/silicon_creator/lib/drivers/spi_device_unittest.cc

```diff
@@ -198,6 +196,8 @@ TEST_P(CmdGetTest, CmdGet) {
                                      std::numeric_limits<uint32_t>::max());
   std::memcpy(payload_area.data(), GetParam().payload.data(),
               GetParam().payload.size());
+  EXPECT_ABS_READ32(base_ + SPI_DEVICE_UPLOAD_STATUS2_REG_OFFSET,
+                    GetParam().payload.size());
```

Comment:
@arunthomas @alphan 
I also have to revise this to pass the meson test (Thanks @timothytrippel for letting me know how to run the test!)
` cd build-out && meson test opentitan:mask_rom / sw_silicon_creator_lib_driver_spi_device_unittest --print-errorlogs`

Please take a look and revise later.